langchain/src/evaluation/agents/prompt.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Prompt for trajectory evaluation chain.
+ */
+
+import {
+  AIMessagePromptTemplate,
+  ChatPromptTemplate,
+  HumanMessagePromptTemplate,
+  SystemMessagePromptTemplate,
+} from "../../prompts/index.js";
+
+const EVAL_TEMPLATE = `An AI language model has been given access to the following set of tools to help answer a user's question.
+
+The tools given to the AI model are:
+[TOOL_DESCRIPTIONS]
+{toolDescriptions}
+[END_TOOL_DESCRIPTIONS]
+
+The question the human asked the AI model was:
+[QUESTION]
+{question}
+[END_QUESTION]{reference}
+
+The AI language model decided to use the following set of tools to answer the question:
+[AGENT_TRAJECTORY]
+{agentTrajectory}
+[END_AGENT_TRAJECTORY]
+
+The AI language model's final answer to the question was:
+[RESPONSE]
+{answer}
+[END_RESPONSE]
+
+Let's do a detailed evaluation of the AI language model's answer step by step.
+
+We consider the following criteria before giving a score from 1 to 5:
+
+i. Is the final answer helpful?
+    ii. Does the AI language use a logical sequence of tools to answer the question?
```

Comment:
Maybe this is just git rendering, but looks like there're some unwanted indentations here?